js/src/views/Signer/store.js

```diff
@@ -41,8 +41,9 @@ export default class SignerStore {
     this.balances = Object.assign({}, this.balances, balances);
   }
 
-  @action setLocalHashes = (localHashes) => {
-    if (!isEqual(localHashes, this.localHashes)) {
+  @action setLocalHashes = (localHashes = []) => {
+    // Use slice to make sure they are both Arrays (MobX uses Objects for Observable Arrays)
+    if (!isEqual(localHashes.slice(), this.localHashes.slice())) {
```

Comment:
Not sure which one is more performant from a MobX perspective - `.peek()` or `.slice()` (Typically do use the former when it is a known (i.e. on `this.`) value